TestProjects/UniversalGraphicsTest_Foundation/Assets/Scenes/120_PostProcessingRenderUICustomRenderer/120_PostProcessingRenderUICustomRenderer_Renderer.asset

```diff
@@ -15,7 +15,7 @@ MonoBehaviour:
   m_RendererFeatures:
   - {fileID: 4450485241359397263}
   m_RendererFeatureMap: 
-  postProcessData: {fileID: 11400000, guid: 41439944d30ece34e96484bdb6645b55, type: 2}
+  postProcessData: {fileID: 0}
   xrSystemData: {fileID: 11400000, guid: 60e1133243b97e347b653163a8c01b64, type: 2}
   shaders:
     blitPS: {fileID: 4800000, guid: c17132b1f77d20942aa75f8429c0f8bc, type: 3}
```

Comment:
Is this renderer being used in this project?

Reply:
used inthe scenes:
120
121
122

Reply:
removed the renderer from those scenes using the default renderer